posts/2024-07-30-24.0.0.8-beta.adoc

```diff
@@ -0,0 +1,816 @@
+
+To use the `facesContainer-4.1` feature, add the following code to your `server.xml` file:
+
+[source,xml]
+----
+<featureManager>
+   <feature>facesContainer-4.1</feature>
+</featureManager>
+----
+
+For more information, see the following resources:
+
+* link:https://openliberty.io/docs/latest/reference/feature/facesContainer-4.0.html[Jakarta Faces Container 4.0] feature
+* link:https://openliberty.io/blog/2024/07/02/24.0.0.7-beta.html#faces41[Faces 4.1 beta blog announcement]
+* link:https://jakarta.ee/specifications/faces/4.1/[Faces 4.1 Javadocs and specification document].
+
+
+// DO NOT MODIFY THIS LINE. </GHA-BLOG-TOPIC>
+
+// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // //
+// Blog issue: https://github.com/OpenLiberty/open-liberty/issues/28769
+// Contact/Reviewer: volosied,pnicolucci
+// // // // // // // //
+
+[#websocket]
+== Improve asynchronous handling with Jakarta WebSocket 2.2
+
+The Jakarta WebSocket 2.2 (`websocket-2.2`) feature enables communication for endpoints by using the Websocket protocol. It is an implementation of the Jakarta EE 11 Websocket 2.2 Specification.
+
+Although very few specification changes were listed for the 2.2 release, the main change from 2.1 is the introduction of the link:https://jakarta.ee/specifications/websocket/2.2/apidocs/server/jakarta/websocket/sendresult#getSession()[getSession()] method on the `SendResult` class. This method enables you to retrieve the original session that was used in the asynchronous remote call. This capability is useful for broadcasting messages to numerous listening clients.
+
+The next beta release of websocket-2.2 will limit the link:https://jakarta.ee/specifications/websocket/2.2/apidocs/server/jakarta/websocket/onmessage#maxMessageSize()[MaxMessageSize] to `Integer#MAX_VALUE` for the `OnMessage` annotation to avoid type errors.
+
+To enable this feature, add the following code to your `server.xml` file.
+
+[source,xml]
+----
+ <featureManager>
+    <feature>websocket-2.2</feature>
+</featureManager>
+----
+
+For more information, such as the Javadocs, specification documents, and other details,see link:https://jakarta.ee/specifications/websocket/2.2/[Jakarta WebSocket 2.2].
+
+// DO NOT MODIFY THIS LINE. </GHA-BLOG-TOPIC>
+
+// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // //
+// Blog issue: https://github.com/OpenLiberty/open-liberty/issues/28921
+// Contact/Reviewer: Riva-Tholoor-Philip, jhanders34
+// // // // // // // //
+
+[#jpa]
+== Use Jakarta Persistence with improved type safety and other enhancements in Jakarta Persistence 3.2
+
+The Jakarta Persistence API is a richly featured API which provides object-model approach to persisting, fetching, and modifying data stored on a relational database system.

```

Comment:
```suggestion
The Jakarta Persistence feature provides a rich API which includes an object-model approach to persist, fetch, and modify data stored on a relational database system.
```
If you wanted to keep the verbs with `ing`, then I think the `to` should change to `for`.